Makefile

```diff
@@ -24,7 +24,56 @@ overfit:
 smalldata:
 	$(PYTEST) tests -v -m "smalldata"
 
-SNLI:
-	$(PYTEST) tests -v -m SNLI
+
+# ===========
+# SNLI
+# ===========
+
+snli:
+	./data/SNLI/download.sh
+snli-esim:
+	 $(PYTHON) bin/jack-train.py with \
+		loader=snli \
+		train=data/SNLI/snli_1.0/snli_1.0_train.jsonl \
+		dev=data/SNLI/snli_1.0/snli_1.0_dev.jsonl \
+		test=data/SNLI/snli_1.0/snli_1.0_test.jsonl \
+		model=esim_snli_reader
+
+
+# ===========
+# SQuAD
+# ===========
+#
+squad:
+	./data/SQuAD/download.sh
+
+glove:
+	./data/GloVe/download.sh
+
+squad-fastqa:
+	#$(PYTHON) bin/jack-train.py with config='./conf/fastqa.yaml'
+	$(PYTHON) bin/jack-train.py with \
```

Comment:
embeddings need to be defined here. FastQA only works when embeddings are provided. Just use the config statement instead. The README shows how to use the commandline instead